### DISCUSSION TITLE
Introducing the Official Nazi Zombies: Portable Documentation Page!

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#432] [Announcements]
Over the last week, I've slowly been building and writing enough material to justify the creation for official documentation. Tonight, I'm ready to release what I've come up with so far! The page is now live at https://docs.nzp.gay/.

While still lacking a lot of content, as of now it includes a lot of material relevant to map creation, in particular elements I've deemed "ready enough" to have comprehensive pages written about, and elements some community members may have trouble with without such documentation. Eventually build scripts will be hosted here on this GitHub organization to allow community collaboration!

As always, the standard https://nzp.gay will still link to the WebGL build of the project!

Thanks as always to my [Patrons](https://patreon.com/cypressimplex), whose donations have allowed me to pay for hosting via Neocities and sustain the nzp.gay domain now that I am in ownership!:
* K Z
* mochi
* Rinse a Gateau Mandate
* Tyler Young
* Benjamin Engelhardt
* botdog
* Derped Crusader
* Jackson Clayton
* Ryan Baldwin
* LEXX
* Mrs. Lavender

**Comment by Conmiester** [comment]
Since I've been really tempted to create a few custom maps for NZP, this will come handy despite there's more to come later.

**Comment by pahntomforces-guy** [comment]
how can i make maps

**Reply by MotoLegacy** [reply]
I really wish people were capable of reading.

**Reply by pahntomforces-guy** [reply]
i am, but i dont
 know where the link is

**Reply by MotoLegacy** [reply]
First paragraph:

"Tonight, I'm ready to release what I've come up with so far! The page is now live at https://docs.nzp.gay/."

**Reply by pahntomforces-guy** [reply]
whoops, i misread sorry
